devices.js

```diff
@@ -1519,14 +1519,54 @@ const devices = [
         model: '73739',
         vendor: 'Sylvania',
         description: 'LIGHTIFY LED RGBW BR30',
-        extend: generic.light_onoff_brightness_colortemp_colorxy,
+        supports: generic.light_onoff_brightness_colortemp_colorxy.supports,
+        toZigbee: generic.light_onoff_brightness_colortemp_colorxy.toZigbee.concat([
+            tz.osram_remember_state,
+            tz.osram_set_transition,
+        ]),
+        fromZigbee: generic.light_onoff_brightness_colortemp_colorxy.fromZigbee.concat([
+            fz.ignore_genIdentify_change,
+            fz.ignore_diagnostic_change,
+            fz.ignore_genscenes_change,
+        ]),
+        configure: (ieeeAddr, shepherd, coordinator, callback) => {
+            const device = shepherd.find(ieeeAddr, 3);
+            const cfgOnOff = {direction: 0, attrId: 0, dataType: 16, minRepIntval: 3, maxRepIntval: 1000, repChange: 1};
+            const cfgLevel = {direction: 0, attrId: 0, dataType: 32, minRepIntval: 3, maxRepIntval: 1000, repChange: 1};
+            const actions = [
+                (cb) => device.bind('genOnOff', coordinator, cb),
+                (cb) => device.foundation('genOnOff', 'configReport', [cfgOnOff], foundationCfg, cb),
+                (cb) => device.bind('genLevelCtrl', coordinator, cb),
+                (cb) => device.foundation('genLevelCtrl', 'configReport', [cfgLevel], foundationCfg, cb),
```

Comment:
Same as https://github.com/Koenkk/zigbee-shepherd-converters/pull/351#issuecomment-473575855

Reply:
Just reverted those changes. Only left in ignore options and the Osram functions.